views/scss/qtiCreator/_choice.scss

```
@@ -34,6 +40,13 @@
             top: -28px;
             right: 0;
         }
+        .widget-math{
+                @include rtl {
+                    margin-right: 2.15em;
+                    maring-left: 2em;
```

You have a typo here:
```suggestion
                    margin-left: 2em;
```